MSAL/src/native_auth/controllers/sign_in/MSALNativeAuthSignInController.swift

```diff
@@ -301,16 +250,35 @@ final class MSALNativeAuthSignInController: MSALNativeAuthTokenController, MSALN
                 continuationToken: continuationToken,
                 scopes: scopes
             )
+        case .strongAuthRequired(let newContinuationToken):
+            MSALLogger.log(level: .info, context: context, format: "Strong authentication required.")
```

Comment:
Is "strong auth needed" the same as MFA? https://microsofteur.sharepoint.com/:w:/r/teams/PlatformExtensibility/_layouts/15/Doc.aspx?sourcedoc=%7B3721D75C-F2B9-41CB-890B-2C185685E171%7D&file=Native%20Auth%20API%20MFA%20-%20How.docx&action=default&mobileredirect=true&share=IQFc1yE3ufLLQYkLLBhWheFxATvCOsmZuMhKKOi3ydVFBLE lists it as 2 separate API responses, though I don't really understand the difference between the two.

Reply:
I think MFA_required and strong auth required are the same response but let me double check with the API team.

Reply:
Clarified with API team. No need to change the log in this case